.github/workflows/bump-deps.yaml

```diff
@@ -0,0 +1,64 @@
+name: Bump DEPS
+on:
+  workflow_dispatch:
+    inputs:
+      dep:
+        description: "Dependency"
+        required: true
+        type: string
+        default: "vendor/web-discovery-project"
+      ref:
+        description: "New ref for dependency"
+        required: true
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  auto-bump:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: "0"
+
+      - name: auto-bump
+        run: |
+          set -x
+
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          git config user.name "${{ github.actor }}"
+          git config push.default simple
```

Comment:
Can we have signed commits here?

Reply:
Interesting. Looking into it

Reply:
@oajara is working on signing for bots, but it's not merged yet and focuses on Jenkins.

Reply:
I got this working. A signed commit can be found in this test PR
https://github.com/brave/brave-core/pull/16135
This is by creating the commit directly via GH API and not `git` cli